bindgen/spec.yml

```diff
@@ -1027,6 +1033,7 @@ classes:
       get:
         - sig: '(key: StringData) const -> Obj'
           cppName: get<Obj> # TODO can't distinguish null from missing
+          suffix: obj
```

Comment:
Technically, we don't need to do this for `Dictionary`, since it doesn't need the API compat with `Results` or other collections. However, it seemed worth maintaining the pattern that `getObj()` is for `Obj` and `getAny()` is for everything else.